.storybook/preview-head.html

```diff
@@ -7,5 +7,12 @@
     --mb-color-focus: #cbe2f7;
     --mb-color-bg-dark: #93a1ab;
     --mb-color-bg-light: #f9fbfc;
+
+    /*
+      Theming-specific CSS variables.
+      These CSS variables are not part of the core design system colors.
+    **/
+    --mb-color-bg-dashboard: var(--mb-color-bg-white);
+    --mb-color-bg-dashboard-card: var(--mb-color-bg-white);
```

Comment:
I think this will be needed for visual-testing dashboards later, but I can remove for now.